src/statement.rs

```diff
@@ -191,6 +191,27 @@ where
         Ok(())
     }
 
+    /// Execute the current statement with input and output paramters
```

Comment:
I think it is confusing to call the return `output parameters`, it's better to say that it returns a single row

Reply:
Done